<p>The sum of the squares of the first ten natural numbers is,</p>
$$1^2 + 2^2 + ... + 10^2 = 385.$$
<p>The square of the sum of the first ten natural numbers is,</p>
$$(1 + 2 + ... + 10)^2 = 55^2 = 3025.$$
<p>Hence the difference between the sum of the squares of the first ten natural numbers and the square of the sum is $3025 - 385 = 2640$.</p>
<p>Find the difference between the sum of the squares of the first one hundred natural numbers and the square of the sum.</p>


In [1]:
# Sum of squares of the first 100 natural numbers
sum_of_squares = sum([i**2 for i in range(1, 101)])

# Square of the sum of the first 100 natural numbers
square_of_sum = sum(range(1, 101))**2

# Difference between the sum of the squares and the square of the sum
difference = square_of_sum - sum_of_squares

print("The difference between the sum of the squares of the first one hundred natural numbers and the square of the sum is", difference)


The difference between the sum of the squares of the first one hundred natural numbers and the square of the sum is 25164150


To find the difference between the sum of the squares of the first one hundred natural numbers and the square of the sum, we can use the following formulas:

The sum of the squares of the first n natural numbers is given by the formula [n(n+1)(2n+1)]/6 1.
The square of the sum of the first n natural numbers is given by the formula [n(n+1)/2]^2 2.
Substituting n = 100 in the above formulas, we get:

The sum of the squares of the first one hundred natural numbers is 338350 1.
The square of the sum of the first one hundred natural numbers is 25502500 2.
Therefore, the difference between the sum of the squares of the first one hundred natural numbers and the square of the sum is:

25502500 - 338350 = 25164150